### DISCUSSION TITLE
Method of rounding in Deaths by AgeGroup

### DISCUSSION THREAD

**Discussion by freedomlives** [Institut-Zdravotnych-Analyz/covid19-data#43] [Q&A] [answered]
How is rounding done with the ages of victims? e.g. if someone who dies is 68, is the age in the data going to be presented as 65 or 70? (Round up vs. round down)
Or is it that those from 62.5 - 67.5 years are listed as 65, then 67.5 - 72.5 is 70, etc?

**Comment by MartinHBA** [comment, marked as answer]
similarly they do it for PCR, so I think this screenshot makes it clear for you 
![image](https://user-images.githubusercontent.com/49184077/106368869-5aaf4680-634d-11eb-9dcb-d2f8c7e1699c.png)

**Reply by freedomlives** [reply]
Thanks! I noticed the same thing later, looking at the testing data, that there was a AgeGroup of 0, so only logical explanation is how statistical office makes age brackets as well - 0-4, 5-9, 60-64.
Interestingly, I noticed that there was one woman testing positive in the 110-114 Age Group, but no one dying in that age group (at least not from COVID19). I would like to get some tips from that person, how to live so long!
